h/browser/chrome/lib/sidebar-injector.js

```diff
@@ -120,8 +120,9 @@ function SidebarInjector(chromeTabs, dependencies) {
         return executeScriptFn(tab.id, {
             code: toIIFEString(detectContentType)
           }).then(function (frameResults) {
```

Comment:
I know it's not new in this PR, but a nice feature of promises is that this kind of nesting isn't needed. This is what we've got here:

```javascript
foo(x).then(function (y) {
  return bar(y).then(function (z) {
    return baz(z);
  });
});
```

which is much clearer as:

```javascript
foo(x)
  .then(bar)
  .then(baz)
```